spec/fixtures/files/xml/anthro/claim1.xml

```diff
@@ -27,8 +27,7 @@
     <dateArrangedForTransferNote>pickup/transfer</dateArrangedForTransferNote>
     <objectsMarkedAsDeaccessionedNote>marked as deaccessioned</objectsMarkedAsDeaccessionedNote>
     <workingTeamNotifiedNote>NAGPRA team notified
-￼
-</workingTeamNotifiedNote>
```

Comment:
NICE WORK trying out the object replacement character, @kevin-kariuki! 🥇  
The mapper strips off any newline character that isn't followed by anything else, so I removed the newline after that character in the XML to get this to pass.